events_structs.go

```diff
@@ -39,11 +42,14 @@ func (ev *Event) When() string {
 }
 
 func (ev *Event) Who() string {
+	if ev.User.Email == "deploy@scalingo.com" {
+		return "Scalingo Operator"
+	}
```

Comment:
@Soulou I updated the `Who` method to match what is asked in the related issue. But not that it also impacts other event types. For instance in case of a crash or an alert. Is it OK for you?

Reply:
Could we keep something more generic like Scalingo Platform? (It seems to me it's already what is in the event no?)

It's weird to say an operator crashes a container

Reply:
The current output is `<scalingo-platform (deploy@scalingo.com)>`. I wanted to follow what is asked on the related issu but I agree that this feels a bit odd. Or maybe I could try to modify it only for the run event?

Reply:
For the run event I agree, for these rest it's misleading..

Reply:
OK that's what is included in the commit 4782dab